node/setup.go

```diff
@@ -46,7 +46,7 @@ const readHeaderTimeout = 10 * time.Second
 // SHA256 checksum.
 type ChecksummedGenesisDoc struct {
 	GenesisDoc     *types.GenesisDoc
-	Sha256Checksum []byte
+	Sha256Checksum [32]byte
```

Comment:
```suggestion
	Sha256Checksum [tmhash.Size]byte
```
since we use `tmhash.Sum` we use its size, I know we named it `Sha256Checksum` but probably should have called it only `Checksum` to allow changes in the checksum in the future

Reply:
happy to rename to checksum if thats preferred

Reply:
if you guys prefer this happy to do it.